lib/packwerk/configurations/README.md

```diff
@@ -0,0 +1,3 @@
+### Configurations
+
+Includes validators which are outside of primary `check` runs.
```

Comment:
I would probably go with the comment in https://github.com/Shopify/packwerk/pull/144/files#diff-5ca9dcd54fd0b9eeef94e00ba6adc7822dc83e5dd2bfb9e8d96fd3c356de8b8aR10-R11